supply/lib/supply/setup.rb

```diff
@@ -43,45 +42,52 @@ def store_metadata(listing)
     def download_images(listing)
       UI.message("🖼️  Downloading images (#{listing.language})")
 
-      # We cannot download existing screenshots as they are compressed
-      # But we can at least download the images
       require 'net/http'
 
-      IMAGES_TYPES.each do |image_type|
-        if ['featureGraphic'].include?(image_type)
-          # we don't get all files in full resolution :(
-          UI.message("📵  Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphic. Please copy your feature graphic to `metadata/android/#{listing.language}/images/featureGraphic.png`")
-          next
-        end
+      allowed_imagetypes = [Supply::IMAGES_TYPES, Supply::SCREENSHOT_TYPES].flatten
 
+      allowed_imagetypes.each do |image_type|
         begin
           UI.message("Downloading `#{image_type}` for #{listing.language}...")
 
-          url = client.fetch_images(image_type: image_type, language: listing.language).last
-          next unless url
+          urls = client.fetch_images(image_type: image_type, language: listing.language)
+          next if urls.nil? || urls.empty?
+
+          image_counter = 1 # Used to prefix the downloaded files, so order is preserved.
+          urls.each do |url|
+            url_params = url.match("=.*")
+            if !url_params.nil? && url_params.length == 1
+              UI.verbose("Initial URL received: '#{url}'")
+              url = url.gsub(url_params.to_s, "") # Remove everything after '=' (if present). This ensures webp is converted to png/jpg as well.
+              UI.verbose("Removed params ('#{url_params}') from the URL")
+              UI.verbose("URL after removing params: '#{url}'")
+            end
+
+            url = "#{url}=s0" # '=s0' param ensures full image size is returned (https://github.com/fastlane/fastlane/pull/14322#issuecomment-473012462)
+
+            if IMAGES_TYPES.include?(image_type) # IMAGE_TYPES are stored in locale/images location
+              path = File.join(metadata_path, listing.language, IMAGES_FOLDER_NAME, image_type.to_s)
+            else # All other screenshot types goes under their respective folders.
+              path = File.join(metadata_path, listing.language, IMAGES_FOLDER_NAME, image_type, "#{image_counter}_#{listing.language}")
+            end
+
+            p = Pathname.new(path)
+            FileUtils.mkdir_p(p.dirname.to_s)
+
+            path = "#{path}.#{FastImage.type(url)}"
+            File.write(path, Net::HTTP.get(URI.parse(url)))
```

Comment:
Currently this doesn't work on Windows at all, just produces black images. With this change it works fine:

```suggestion
            File.binwrite(path, Net::HTTP.get(URI.parse(url)))
```